src/app/components/connect-addon-modal/connect-addon-modal.html

```diff
@@ -0,0 +1,82 @@
+<div id="connect-addon-modal">
+  <div class="modal-header">
+    <div class="close" type="button" aria-label="Close" ng-click="$ctrl.close()">×</div>
+    <h3 class="modal-title"
+        translate="mno_enterprise.webhook.o_auth.providers.xero.connect_to"
+        translate-values="{appname: $ctrl.app.name}"></h3>
+  </div>
+
+  <!-- Link -->
+  <div ng-show="!$ctrl.hasLinked && !$ctrl.hasChosenEntities">
+    <ul class="breadcrumb">
+      <li>{{"mno_enterprise.templates.components.addon_connect.link_account.title" | translate}}</li>
+      <li><a href="#">{{"mno_enterprise.templates.components.addon_connect.entities.select" | translate}}</a></li>
+      <li><a href="#">{{"mno_enterprise.templates.components.addon_connect.sync.start_sync" | translate}}</a></li>
+    </ul>
+
+    <div class="modal-body">
+      <h4 class="text-center">{{"mno_enterprise.templates.components.addon_connect.link_account.link" | translate:{ appname: $ctrl.app.app_name} }}</h4>
+      <div class="top-buffer-1"></div>
+
+      <form sf-schema="$ctrl.schema" sf-form="$ctrl.form" sf-model="$ctrl.model" ng-submit="$ctrl.submit(setupForm)" name="setupForm" ng-if="!$ctrl.isFormLoading"></form>
+      <mno-loader ng-if="$ctrl.isFormLoading"></mno-loader>
+
+    </div>
+  </div>
+
+  <!-- Chose Entities -->
+  <div ng-show="$ctrl.hasLinked && !$ctrl.hasChosenEntities">
+    <ul class="breadcrumb">
+      <li><a href="#">{{"mno_enterprise.templates.components.addon_connect.link_account.title" | translate}}</a></li>
+      <li>{{"mno_enterprise.templates.components.addon_connect.entities.select" | translate}}</li>
+      <li><a href="#" ng-click="$ctrl.forceSelectEntities()">{{"mno_enterprise.templates.components.addon_connect.sync.start_sync" | translate}}</a></li>
+    </ul>
+
+    <div class="modal-body">
+      <h4 class="text-center">{{"mno_enterprise.templates.components.addon_connect.entities.chose_entities" | translate}}</h4>
```

Comment:
Please center this title:
![screen shot 2017-11-13 at 10 32 07 am](https://user-images.githubusercontent.com/3941996/32704727-f603f4b6-c85d-11e7-959e-a7eaa602d927.png)